### PR TITLE
chore: Enable MD030

### PR DESCRIPTION
**Pull request by nschonni**
Should probably wait till #12040, #12041, #12042 have landed

### CODE REVIEW

**Comment by BillWagner** [comment]
Am I correct that this should wait until #12040 is merged?

**Comment by nschonni** [comment]
@BillWagner this should be good to land now

**Review by BillWagner — APPROVED** [review]
Thanks again for helping us in such significant ways.

This is ready to :shipit:

**Pull request merged**
